### PR TITLE
Updated links to point to new website

### PR DESCRIPTION
**Pull request by yaichenbaum**

### CODE REVIEW

File: Files/ViewModels/SettingsViewModels/AboutViewModel.cs [hunk 1]
```diff
@@ -57,7 +57,7 @@ private async void ClickAboutFeedbackItem(ItemClickEventArgs e)
                     break;
 
                 case "Documentation":
-                    await Launcher.LaunchUriAsync(new Uri(@"https://files-community.github.io/docs"));
+                    await Launcher.LaunchUriAsync(new Uri(@"https://files.community/docs"));
```
Comment (winston-de) [inline Files/ViewModels/SettingsViewModels/AboutViewModel.cs:60]: Should this be a constant to make it easier to update in the future?

**Comment by winston-de** [comment]
Also the links don't seem to work, is the site just not up yet?

File: Files/ViewModels/SettingsViewModels/AboutViewModel.cs (hunk 1 above)
Comment (yaichenbaum) [reply Files/ViewModels/SettingsViewModels/AboutViewModel.cs:60]: Sounds like a good idea, where would you store the constant?

**Comment by yaichenbaum** [comment]
> Also the links don't seem to work, is the site just not up yet?

The site will be going live with v2.

**Review by winston-de — APPROVED** [review]

**Pull request merged**